lib/registries/addon/overview/-components/overview-topbar/template.hbs

```diff
@@ -0,0 +1,100 @@
+<div data-test-topbar-states local-class='LeftActionsContainer'>
+    <RegistriesStates @registration={{@registration}} />
+</div>
+<div data-test-topbar-share-bookmark-fork
+    local-class='RightActionsContainer'
+>
+    <ResponsiveDropdown as |dd|>
+        <dd.trigger
+            data-test-forks-dropdown-button
+            data-analytics-name='Expand fork dropdown'
+            local-class='Action'
+        >
+            <FaIcon
+                local-class='Icon'
+                @icon='code-fork'
+            />
+            <BsTooltip
+                @placement='top'
+                @triggerEvents='hover'
+            >
+                {{t 'registries.overview.tooltips.fork'}}
+            </BsTooltip>
+        </dd.trigger>
+        <dd.content>
+            <div data-test-forks-dropdown-options local-class='ForksMenu'>
+                <div local-class='ForksMenu__option'>
```

Comment:
These links don't take up the full width of the menu, which is unexpected (and inconsistent with the "share" menu next to it)
![Kapture 2019-05-14 at 16 32 20](https://user-images.githubusercontent.com/6776190/57782032-67af8080-76f9-11e9-81c1-d24389437d2e.gif)

Might be cleanest to give `dd.content` a local class with `flex-direction: column`, then get rid of these wrapper `div`s